src/execution_plan/ops/op.c

```
@@ -39,32 +75,33 @@ void OpBase_Init
+	op->_consume = consume;          // op's consume function
 	op->toString = toString;
+
+	op->init  = (init)  ? init  : _OpBase_init_noop;
+	op->reset = (reset) ? reset : _OpBase_reset_noop;
 }
 
 inline Record OpBase_Consume
 (
 	OpBase *op
 ) {
+	ASSERT(op != NULL);
```

cppcheck (2.10-2)

[error] Null pointer dereference

(nullPointer)